tests/testapps/CGCatalog/CGCatalog/CGPathAddLineToPointViewController.m

```diff
@@ -47,6 +47,10 @@ - (void)loadView {
         CGPathAddLineToPoint(thepath, NULL, 300, 100);
         CGPathAddLineToPoint(thepath, NULL, 235, 100);
 
+        CGPoint points[] = { { 300, 200 }, { 400, 200 }, { 400, 300 }, { 300, 300 }, { 300, 200 } };
+
+        CGPathAddLines(thepath, NULL, points, 5);
```

Comment:
Oh woops, didn't mean to check this in. I was verifying the behavior of addlines. Ugly star is sad.

---
In reply to: [83963476](https://github.com/Microsoft/WinObjC/pull/1138#discussion_r83963476) [](ancestors = 83963476)